features/steps/test_starter/{{ cookiecutter.repo_name }}/pyproject.toml

```diff
@@ -35,7 +35,8 @@ namespaces = false
 package_name = "{{ cookiecutter.python_package }}"
 project_name = "{{ cookiecutter.project_name }}"
 kedro_init_version = "{{ cookiecutter.kedro_version }}"
-tools = "{{ cookiecutter.tools | default('') | string | replace('\"', '\\\"') }}"
+tools = {{ cookiecutter.tools | default('') | string | replace('\"', '\\\"') }}
```

Comment:
why you removed "" there?

Reply:
Removing the double quotes means this is saved as a list instead of a string, which should be helpful for analytics on HEAP, xref #3333